var/spack/repos/builtin/packages/vtk-m/package.py

```diff
@@ -185,6 +185,12 @@ class VtkM(CMakePackage, CudaPackage, ROCmPackage):
         when="@1.6.0:2.1 +cuda ^cuda@12.5:",
     )
 
+    def flag_handler(self, name, flags):
+        if name == "cxxflags":
+            if self.spec.satisfies("%oneapi@2025:"):
+                flags.append("-Wno-error=missing-template-arg-list-after-template-kw")
```

Comment:
This compile error should be fixed in future versions of VTK-m. Should this flag be applied only for version `2.2.0:`?

Reply:
Wouldn't applying it to `2.2.0:` mean 2.2.0 and all versions after, including those future versions that have the issue fixed? What version should we cap this at?

Reply:
Got it! Done:
* https://github.com/spack/spack/pull/47477/commits/605c75de186870714ccfcdf8579e6803a0547603